mrtt-ui/src/styles/themeMui.js

```diff
@@ -10,6 +14,8 @@ const themeMui = createTheme({
       main: theme.color.secondary
     }
   },
+  spacing: (factor) => `${0.5 * factor}rem`,
+
```

Comment:
if you want we can put this spacing function in theme, now that you figured out the mui side of things.  this can also be done later.